form/form.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
Ideally the test related files should be in the same folder as of the unit tests file